packages/flutter/test/material/mergeable_material_test.dart

```diff
@@ -5,6 +5,8 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import '../rendering/mock_canvas.dart';
```

Comment:
We should consider moving this to flutter_test if we're going to use it so much.